content/post/stats_at_work.md

```diff
@@ -7,89 +7,72 @@ date: 2018-07-26T20:35:46-04:00
+Diane: They can. Basically given a significance threshold and a sample size, our test is able to detect some level of difference between A and B. If we have a large enough sample size, then we can detect very small differences. It's kinda like the precision of our test. But this sometimes gives p-values a bad wrap, because you can make almost any test seem significant with enough data. That's why I asked you at the beginning what difference in conversion rate you care about testing for.   
+Marty: And we said that we care about differences around 0.5%. 
 
-DS: Yup. And the sample size.  
-MA: Right that was the next thing I was curious about. How did you know what the right sample size was?
+Diane: Right, so I set up the test to be able to identify a difference of 0.5%. When we collected enough samples, we stopped the test and looked at the result.  
+Marty: Which showed that the p-value was less than 5%, and hence a difference existed between A and B.
 
-DS: Yeah, so that's all about figuring out how many users we needed to be confident in our conclusion.  
-MA: Is that the confidence interval?
+Diane: Yup.  
+Marty: So is there a formula for figuring our the right sample size?
 
-DS: Sort of. It's related. The confidence interval is more about telling us the interval around our estimate. Remember, we're estimating the difference between A's conversion rate and B's conversion rate.  
-MA: How does sample size fit in? Is there a formula for that?
+Diane: It's all one formula actually! Given a significance threshold, an effect size and a power level, I can calcu-  
+Marty: Wait what's power?
 
-DS: It's all one formula actually! Given a significance threshold, an effect size and a power level, I can calcu-  
-MA: Wait what's power?
+Diane: Oh sorry. Power is a term that's often overlooked because it's usually baked into the calculations. Power is sometimes used in place of sample size because they are very much linked, even though they are different concepts. The power of a test is the probability of detecting a difference when it exists. So power is a good thing. And we manipulate the power using sample size. Bigger sample, more power.  
+Marty: With you.
 
-DS: Oh sorry. Power is a term that's often overlooked because it's usually baked into the calculations. Power is sometimes used in place of sample size because they are very much linked, even though they are different concepts. Let's take a step back. Recall there are two things we want to check for in this A/B test. First, that A is truly bigger than B by at least the amount we care about (the effect size). Second, we want to be confident in our conclusion.  
-MA: And that we have a big enough sample size. 
+Diane: Recall there are two things we want to check for in this A/B test. First, that A is truly bigger than B by at least the amount we care about, our desired effect size. Second, that we collect a big enough sample to be able to detect that difference.  
```

Comment:
I'm not sure that this is the correct use of "effect size" in the scenario we're discussing. 

In this example, I think, the way we're using effect size goes something like this:

> We will detect *true* effect sizes that are > 0.5%  eighty percent of the time with 95% confidence

That is, for *true* effects that are smaller than 0.5% we will not have our target 80% power, but we may still detect them as being significant! The implied NHT we're running here is still using H_0: A - B = 0. If we wanted to be confident that A - B > 0.005 then we'd need to do the math a bit differently.

Does this make sense?

Reply:
Ahh good catch. The H0 is A = B and HA is A != B, not A > B + 0.005. Helps to write it out. We're  interested in detecting true differences >= 0.5%, not just any difference.